examples/opengl_underlay/build.rs

```diff
@@ -0,0 +1,7 @@
+// Copyright © SixtyFPS GmbH <info@sixtyfps.io>
+// SPDX-License-Identifier: (GPL-3.0-only OR LicenseRef-SixtyFPS-commercial)
+
+fn main() {
+    let config = sixtyfps_build::CompilerConfiguration::default().with_style("fluent".into());
```

Comment:
why do we need to set the style?

Reply:
Normally it should default to native only if the Qt backend is in the dependency tree. And it is ok to use the native style with the GL backend at runtime.

If you had an error maybe the style detection didn't work. There is a bug that you need to recompile twice after changing the feature set because the build script is not re-run

Reply:
I had some local issues but it works indeed, so this is now removed :)